src/js/common/helpers/api.js

```diff
@@ -15,10 +15,12 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
     ? [baseUrl, resource].join('')
     : resource;
 
+  const Authorization = sessionStorage.userToken && `Token token=${sessionStorage.userToken}`;
+
   const defaultSettings = {
     method: 'GET',
     headers: {
-      Authorization: `Token token=${sessionStorage.userToken}`,
+      Authorization,
       'X-Key-Inflection': 'camel'
```

Comment:
I'm adding a check here because otherwise if sessionStorage.userToken is `undefined`, then the authorization header is set to `Token token=undefined`. Then our mock API registers the requests to that instead of `_global`.